packages/jaeger-ui/src/components/App/Page.js

```diff
@@ -52,26 +56,29 @@ export class PageImpl extends React.Component<Props> {
   }
 
   render() {
+    const { embedded } = this.props;
+    const contentCls = cx({ 'Page--content': !embedded });
     return (
       <div>
         <Helmet title="Jaeger UI" />
         <Layout>
-          {!isEmbed(this.props.search) && (
+          {!embedded && (
             <Header className="Page--topNav">
               <TopNav />
             </Header>
           )}
-          <Content className={!isEmbed(this.props.search) && 'Page--content'}>{this.props.children}</Content>
+          <Content className={contentCls}>{this.props.children}</Content>
         </Layout>
       </div>
     );
   }
 }
 
 // export for tests
-export function mapStateToProps(state: { router: { location: Location } }) {
+export function mapStateToProps(state: ReduxState) {
+  const { embedded } = state;
```

Comment:
couldn't this `embedded` flag come from router? or are there other ways of enabling beyond query params?

Reply:
I tested this and you can pass uiEmbed=v0 in query parameters.